### ISSUE TITLE
conda channels ignored with biocontainers

### ISSUE THREAD

**Issue by bernt-matthias** [galaxyproject/planemo#1092]
It seems that `--conda_channels` is ignored when `--biocontainers` is enabled

**Comment by jmchilton** [comment]
Yeah - conda channels only applies to non-container conda resolution.

**Comment by bernt-matthias** [comment]
But as far as I see conda is called if no container exists already, or?

**Comment by bernt-matthias** [comment]
From @mvdbeek on gitter:

> If you’re tesing with mulled builder we’d need a new galaxy-tool-util package with galaxyproject/galaxy#11966, so you could set DEFAULT_MULLED_CONDA_CHANNELS
